astropy/cosmology/_io/mapping.py

```diff
@@ -411,7 +429,9 @@ def to_mapping(
     return m if rename is None else _rename_map(m, rename)
 
 
-def mapping_identify(origin, format, *args, **kwargs):
+def mapping_identify(
+    origin: str, format: Literal["mapping"] | None, *args: object, **kwargs: object
```

Comment:
The annotation for `format` is inconsistent with `cosmology_identify`. Both approaches seem fine to me, but we should stick to one.

Reply:
str | None

Reply:
corrected.